frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -386,168 +386,184 @@ class Gallery extends React.Component {
     return state.imagesPreviews[state.fileSelected].rawFileUrl
   }
 
+  onMouseMove = () => {
+    clearInterval(this.mouseMoveTimeout)
+    if (this.state.displayLightbox) {
+      document.getElementsByClassName('ril__toolbar')[0].style.transform = 'translateY(0px)'
+      document.getElementsByClassName('ril__toolbar')[0].style['transition-duration'] = '0.5s'
+    }
+    this.mouseMoveTimeout = setInterval(() => {
+      if (this.state.displayLightbox) {
+        document.getElementsByClassName('ril__toolbar')[0].style.transform = 'translateY(-50px)'
+        document.getElementsByClassName('ril__toolbar')[0].style['transition-duration'] = '0.5s'
+      }
+    }, 2000)
+  }
+
   render () {
     const { state, props } = this
 
     if (state.imagesPreviews[state.fileSelected]) this.lightboxRotation.changeAngle(state.imagesPreviews[state.fileSelected].rotationAngle)
 
     return (
-      <PageWrapper customClass='gallery'>
-        <PageTitle
-          title={state.folderId ? state.fileName : state.content.workspaceLabel}
-          icon={'picture-o'}
-          breadcrumbsList={state.breadcrumbsList}
-        />
-
-        <PageContent>
-          <div className='gallery__action__button'>
-            <button
-              className='btn outlineTextBtn nohover primaryColorBorder'
-              onClick={() => this.onClickSlickPlay(!state.autoPlay)}
-            >
-              <span className='gallery__action__button__text'>
-                {state.autoPlay ? props.t('Pause') : props.t('Play')}
-              </span>
-              <i className={classnames('fa', 'fa-fw', state.autoPlay ? 'fa-pause' : 'fa-play')} />
-            </button>
-
-            <button
-              className='btn outlineTextBtn nohover primaryColorBorder gallery__action__button__rotation__left'
-              onClick={() => this.rotateImg(state.fileSelected, DIRECTION.LEFT)}
-            >
-              <span className='gallery__action__button__text'>{props.t('Rotate 90° left')}</span>
-              <i className={'fa fa-fw fa-undo'} />
-            </button>
+      <div class='gallery__mouse__listener' onMouseMove={this.onMouseMove}>
```

Comment:
I wont agree to this without a very good reason. This is something you must ask me beforehand before I can't be sure to check all these small but important things on a code review.

The purpose of a code review is to check code quality, not architecture. 

There is a PageWrapper component to ensure that every fullscreen app have the same layout
Adding a wrapper might compromise the homogeneity of tracim

I also need a feedback on the React possible built-in features for css modification on mouse move events

Reply:
React provide the the function `onMouseMove` to record the mouse moving in a dom element, so I added a div which wrap all the gallery but yes it isn't the best solution.

I need to add a `div` which is only displayed when the lightbox is open, this `div` need to be fixed and have the size of the screen in order to catch `onMouseMove`.

Css modifications use simple javascript because it is a hack for imageLightBox.